web-app/stories/Page.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Header } from './Header';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
import { Header } from "./Header";
```